src/packages/api/credit-memo/credit-memo-api.ts

```
@@ -0,0 +1,113 @@
+		const token = await this.apiClient.authProvider.getToken();
+
+		// Return the Company Endpoint
+		return `${baseEndpoint}/${token.realmId}`;
+	}
+
+	/**
+	 * Format the Response
+	 * @param response - The Response
+	 * @returns The CreditMemos
+	 */
+	protected formatResponse(response: any): Array<CreditMemo> {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)